code/game/objects/items/storage/pouch.dm

```diff
@@ -255,7 +255,7 @@
 	fill_number = 3
 
 /obj/item/storage/pouch/magazine/large/pmc_rifle
-	fill_type = /obj/item/ammo_magazine/rifle/ap
+	fill_type = /obj/item/ammo_magazine/rifle/explosive
```

Comment:
These actually spawn on PMCs.

Reply:
Yes. That's why I said that in the PR description.

Reply:
It's a balance concern.

Reply:
Upon further thought, you're 100% right. If a single 20 round mag available at roundstart could pretty much win the game for marines every match, giving 3 to PMC's will ensure that sending a distress beacon is just a 25% chance for marine major victory. 

I'll remove it from PMC's when I wake up tomorrow, but make an alternate pouch with it to give to deathsquad standards, as they're supposed to be able to steamroll all opposition.

I also want to make a PR rebalancing ERT spawn chances which would give deathsquad a 5% chance of spawning, but that's for another discussion.

Reply:
Removed from PMC, adding to DS